package.json

```diff
@@ -55,11 +55,10 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@reach/dialog": "^0.16.2",
+    "@radix-ui/react-dialog": "^1.0.4",
     "zustand": "^3.6.9"
   },
   "peerDependencies": {
-    "next": "^12.1.6",
     "react": ">=16",
```

Comment:
Can you update `@types/react` to the latest version please as well? And please add back in the `next` peerDep, it's needed.

Reply:
And `@types/react-dom` as well